.gitignore

```diff
@@ -45,3 +45,8 @@ bwd/
 dist/
 
 src/utils/quicklinks.json
+
+# IDE
+.idea
+package.json
+package-lock.json
```

Comment:
This is not the way to remove the package.json changes. You will need to revert or cherrypick.